crates/ruff_benchmark/benches/red_knot.rs

```diff
@@ -138,7 +138,7 @@ fn benchmark_incremental(criterion: &mut Criterion) {
                 case.fs
                     .write_file(
                         SystemPath::new("/src/foo.py"),
-                        format!("{BAR_CODE}\n# A comment\n"),
+                        format!("{FOO_CODE}\n# A comment\n"),
```

Comment:
I think the real error here is that we write `/src/foo` instead of bar. Because we then touch `bar` and not `foo`. 

The idea is to invalidate a dependency and then re-run checking which should show how well we can cope with local changes.